From 8617bf8d97f5b9b35a76bc07b9f110e037eb273e Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 27 May 2025 08:36:22 +0200
Subject: [PATCH 840/954] fix(stm32mp23): define VOLTD_SCMI_STPMIC2_BUCK

The macros VOLTD_SCMI_STPMIC2_BUCK* weren't defined for STM32MP23,
leading to compilation errors. Use the same definitions as STM32MP25.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git cadc66f13d2b8f292fe9335bf097b8801f5a4cd7 fix(stm32mp23): define VOLTD_SCMI_STPMIC2_BUCK]
Change-Id: I7b796c0b84fd547f9170eb9930fc0eb67ecebe7d
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/462058
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Maxime MERE <maxime.mere@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
---
 plat/st/stm32mp2/services/ca35ss_clk_svc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/services/ca35ss_clk_svc.c b/plat/st/stm32mp2/services/ca35ss_clk_svc.c
index 53ec556c4d..ce6b20986e 100644
--- a/plat/st/stm32mp2/services/ca35ss_clk_svc.c
+++ b/plat/st/stm32mp2/services/ca35ss_clk_svc.c
@@ -32,7 +32,7 @@
 
 #define TIMEOUT_10MS_IN_US		10000U
 
-#if STM32MP25
+#if STM32MP23 || STM32MP25
 #define VOLTD_SCMI_STPMIC2_BUCK1	10U
 #define VOLTD_SCMI_STPMIC2_BUCK		VOLTD_SCMI_STPMIC2_BUCK1
 #elif STM32MP21
-- 
2.53.0

